style.css

```diff
@@ -0,0 +1,220 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    max-width: 1024px;
+    margin: 0 auto;
+    font-family: 'opensans-regular', sans-serif;
```

Comment:
Where are you link this font? And better add one more extra font (like Arial) in font list. Also using the shorthand declaration font: font-style font-variant font-weight font-size/line-height font-family would be good.